tests/unit/lib/foxbox/event-dispatcher_test.js

```diff
@@ -0,0 +1,423 @@
+import EventDispatcher from 'js/lib/foxbox/event-dispatcher';
+
+describe('EventDispatcher >', function () {
```

Comment:
Okay, I'll keep like it's currently, but we can discuss this later, I don't have a preference actually